app/views/layouts/_navbar.html.haml

```diff
@@ -5,30 +5,44 @@
         %span.icon-bar
         %span.icon-bar
         %span.icon-bar
-      - if current_event && current_event.slug
+      - if current_event
         = link_to "#{current_event.name} CFP", event_path(current_event), class: 'navbar-brand'
       - else
-        = link_to "#{Rails.application.class.parent_name}", events_path, class: 'navbar-brand'
+        = link_to "CFP App", events_path, class: 'navbar-brand'
 
     .collapse.navbar-collapse
-      - if user_signed_in?
+      - if current_user
         %ul.nav.navbar-nav.navbar-right
-          - if current_user.proposals.present? || current_user.pending_invitations.present?
-            = render partial: "layouts/nav/proposals_link"
-
-          - if current_event && current_user.reviewer_for_event?(current_event)
-            = render partial: "layouts/nav/reviewer_nav"
-
-          - if current_event && current_user.organizer_for_event?(current_event)
-            = render partial: "layouts/nav/organizer_nav"
-
-          - if current_event && policy(current_event).staff?
+          - if speaker_nav?
+            %li{class: "my-proposals-link"}
+              = link_to proposals_path do
+                %i.fa.fa-file-text
+                %span My Proposals
+
+          - if review_nav?
+            %li{class: "event-proposals-link"}
+              = link_to event_staff_proposals_path(current_event) do
+                %i.fa.fa-file-text
+                %span Review Proposals
+
+          - if program_nav?
+            %li{class: "#{request.path == event_staff_program_path(current_event) ? 'active' : ''}"}
```

Comment:
The inline logic here could live in a helper or decorator method. Something like program_active_css or program_active_class